.gitignore

```diff
@@ -1,3 +1,52 @@
 dist
 node_modules
 zuliprc
+.env
+
+# Adds generated gitignore for Windows and Linux files
```

Comment:
No.

Keep your environment specific gitignores [to your environment](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Only the project-relevant ignores go to the project gitignore.

Reply:
Will be done, thank you i did not know this was possible!